test/transport/transportPolynomials.cpp

```diff
@@ -0,0 +1,87 @@
+#include "gtest/gtest.h"
+
+#include "cantera/transport/TransportFactory.h"
+#include "cantera/thermo/ThermoFactory.h"
+#include "cantera/transport/MixTransport.h"
+
+using namespace Cantera;
+
+class TransportPolynomialsTest : public testing::Test
+{
+public:
+    TransportPolynomialsTest() {
+        phase = newPhase("gri30.cti");
+    }
+
+    void check_viscosity_poly(const std::string speciek, vector_fp visc_coeff_expected, int mode) {
+        MixTransport tran;
+        tran.init(phase, mode);
+        size_t k = phase->speciesIndex(speciek);
+        for (size_t i = 0; i<visc_coeff_expected.size(); i++) { 
+            EXPECT_FLOAT_EQ(tran.viscosityPolynomials()[k][i], visc_coeff_expected[i]);
```

Comment:
This should always be:
```suggestion
            EXPECT_DOUBLE_EQ(tran.viscosityPolynomials()[k][i], visc_coeff_expected[i]);
```